messages/pl.json

```diff
@@ -0,0 +1,63 @@
+{
+  "pickupPointsModal.loading": "Ładuję…",
+  "pickupPointsModal.deliver": "Dostawa",
+  "pickupPointsModal.pickup": "Odbiór",
+  "pickupPointsModal.items": "{itemsAmount, plural, =0 {} one {# produkt} other {# produkty}}",
+  "pickupPointsModal.cancelBackList": "Zobacz wszystkie punkty odbioru",
+  "pickupPointsModal.slaOption": "{name} - {price} - {shippingEstimate}",
+  "pickupPointsModal.free": "Darmowy",
+  "pickupPointsModal.selectPickupPoint": "Wybierz punkt odbioru",
+  "pickupPointsModal.pointDetails": "Szczegóły punktu odbioru",
+  "pickupPointsModal.closeButton": "zamknięty",
+  "pickupPointsModal.list": "Lista",
+  "pickupPointsModal.map": "Mapa",
```

Comment:
All good with `lista` and `mapa`?

Reply:
@garrucho

Reply:
Also waiting for Vlad in Slack. 😕

Reply:
Checked with Vlad and it seems all good.